<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consegna_Materiale</class>
 <widget class="QMainWindow" name="Consegna_Materiale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestione Materiale</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtnMaterialeEsci">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Esci</string>
    </property>
   </widget>
   <widget class="QTableView" name="tblRicerca">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>781</width>
      <height>111</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="grpRicercaCittadino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Ricerca Cittadino</string>
    </property>
    <widget class="QPushButton" name="BtnCerca">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cerca</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtVia">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCognome">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Cognome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblNome">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Via/Viale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCivico">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCivico">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NÂ°</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCognome">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNome">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblRagioneSociale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ragione sociale</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRagioneSociale">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtnPrivato">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Privato</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rBtnAzienda">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Azienda</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="BtnElenco">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>40</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Elenco completo</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpBuste">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Buste</string>
    </property>
    <widget class="QLabel" name="lblRosse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Buste Rosse</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBlu">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Buste Blu</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblVerdi">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Buste Verdi</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblBianche">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Buste Bianche</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxRosse">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxBlu">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxVerdi">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxBianche">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpCalendari">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>120</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Calendari</string>
    </property>
    <widget class="QComboBox" name="comboBoxCalendari">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpMastelli">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Mastelli</string>
    </property>
    <widget class="QLabel" name="lblUmido">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Umido</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxUmido">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblVetro">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Vetro</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxVetro">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpSecchi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Richiesta secchi</string>
    </property>
    <widget class="QComboBox" name="comboBoxSecchi240">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxSecchi1100">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lblSecchi240">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Secchi 240 lt</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSecchi1100">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Secchi 1100 lt</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblUltimaConsegna">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data ultima consegna :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDataUltimaRichiestaDb">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="cmdLinkBtnVerifica">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Verifica</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalva">
    <property name="geometry">
     <rect>
      <x>568</x>
      <y>540</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Salva richiesta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
